Foundation/NSDictionary.swift

```diff
@@ -152,7 +152,7 @@ public class NSDictionary : NSObject, NSCopying, NSMutableCopying, NSSecureCodin
     }
     
     public func copyWithZone(zone: NSZone) -> AnyObject {
-        NSUnimplemented()
+        return NSDictionary(objects: self.allValues, forKeys: self.allKeys as! [NSObject])
```

Comment:
this should just return self, same thing as NSArray; NSMutableDictionary and other subclasses should use the O(n) copy and NSDictionary (immutable) should just be a O(1) return of self